.github/workflows/test_upload_dev_build_to_firebase.yml

```
@@ -0,0 +1,90 @@
+  
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    name: Build
+    runs-on: macOS-latest
+    steps:
+    - name: Checkout Repo
+      uses: actions/checkout@v3
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)